local_build.sh

```diff
@@ -1,16 +1,25 @@
 #!/bin/bash
 
-output=$(go version -m echo $(which golangci-lint))
+output=$(go version -m "$(which golangci-lint)")
 GOARCH=$(grep -oE 'GOARCH=\S+' <<< "$output" | cut -d '=' -f 2)
 GOOS=$(grep -oE 'GOOS=\S+' <<< "$output" | cut -d '=' -f 2)
 tools_version=$(grep 'golang.org/x/tools' <<< "$output" | grep -oE '\bv[0-9]+\.[0-9]+\.[0-9]+\b' | grep -oE '[0-9]+\.[0-9]+\.[0-9]+')
 go_version=$(grep -oE 'go[0-9]+\.[0-9]+\.[0-9]+' <<< "$output" | grep -oE '[0-9]+\.[0-9]+\.[0-9]+')
 
-echo $tools_version
+local_go_version=$(go version | grep -oE '[0-9]+\.[0-9]+\.[0-9]+')
+if [[ "$go_version" != "$local_go_version" ]]; then
+  echo -e "FYI - the go version being used right now ($go_version) is not the same" \
+    "that was used to build golangci-lint ($local_go_version)"
+  echo -e "continuing the build like normal ..."
+fi
+
+echo "$tools_version"
 go mod init github.com/nilaway-plugin
 go mod tidy
-go mod edit -replace golang.org/x/tools=golang.org/x/tools@v$tools_version
+go mod edit -replace golang.org/x/tools=golang.org/x/tools@v"$tools_version"
 go mod tidy
 
-echo building nilaway.so with GOARCH=$GOARCH GOOS=$GOOS tools_version=$tools_version go_version=$go_version ...
```

Comment:
Original line is a little confusing since the GOARCH, GOOS variables are the same as on the user's PC but the `go_version` printed out here has nothing to do with the go version being used to build the plugin.